src/EventStore.ClusterNode/Program.cs

```diff
@@ -35,10 +35,8 @@ internal static class Program {
 			Log.Logger = EventStoreLoggerConfiguration.ConsoleLog;
 			try {
 				var options = ClusterVNodeOptions.FromConfiguration(args, Environment.GetEnvironmentVariables());
-				var logsDirectory = string.IsNullOrWhiteSpace(options.Log.Log)
-					? Locations.DefaultLogDirectory
-					: options.Log.Log;
-				EventStoreLoggerConfiguration.Initialize(logsDirectory, options.GetComponentName(),
+				var logsDirectory = options.GetFullLogsDir();
```

Comment:
This will change the effect of the log line later to add e.g. 127.0.0.1-2113-cluster-node

`[38600, 1,11:27:03.060,INF] "LOGS:"                   "d:\\qa\\log\\127.0.0.1-2113-cluster-node"`

but i dont see a problem with that